rest/data/distro.go

```diff
@@ -136,7 +136,10 @@ func (tc *DBDistroConnector) FindCostByDistroId(distroId string,
 	// DistroCost model.
 	dc := res[0]
 	dc.Provider = d.Provider
-	dc.ProviderSettings = *(d.ProviderSettings)
+	if d.ProviderSettings != nil {
+		dc.ProviderSettings = *(d.ProviderSettings)
+	}
+	// TODO: handle multiple regions in ProviderSettingsList
```

Comment:
This is temporary, just wanted to remind myself to look at it further--how does the cost function work? Will handling multiple regions for cost need to be a separate ticket?